Birthday-2024-Project/Art/UI/Themes/Fauna_2024_birthday_game_theme.tres

```diff
@@ -0,0 +1,236 @@
+[gd_resource type="Theme" load_steps=37 format=3 uid="uid://cp1430xlncsx2"]
+
+[ext_resource type="FontFile" uid="uid://b4qxnl5c1be6s" path="res://Fonts/Baskerville Bold.otf" id="1_4p44h"]
+[ext_resource type="Texture2D" uid="uid://b2pihf1eh3ak" path="res://Art/UI/Button_Hover_128.png" id="2_suw1a"]
```

Comment:
Was using Button_Hover instead because Button_Normal since button text was hard to see with Button_Normal's textured background. Edited Button_Normal to have solid white background like Button_Hover and set back to Button_Normal for normal state